### PR TITLE
refactor: replace unnamed functions with arrow callbacks

### PR DESCRIPTION
**Pull request by Fdawgs**
This PR:

- Replaces unnamed functions with arrow callbacks; [arrow callbacks are already in use in the repo](https://github.com/thegreenwebfoundation/co2.js/blob/4a8737a5b33c4c68ee8581f4c9507e3b0503217a/src/helpers/index.js#L1), and this saves a few bits/bytes by using less loc
- Removes already declared variables in `sustainable-web-design.test.js`; `averageWebsiteInBytes` is declared further up in the scope on line 5
-  Enables the `eslint-plugin-jest` dev dependency rules in `.eslintrc.json` and fixes errors thrown by that plugin

### CODE REVIEW

**Comment by mrchrisadams** [comment]
Thanks for this @Fdawgs. I'd like to leave @drydenwilliams a day or so to look over this before I merge this in, as it touches code he authored.

I'm happy with this though, and this seems pretty uncontroversial. I'll merge in end of Tuesday, if there's no further comment.

**Review by drydenwilliams — APPROVED** [review]

**Pull request merged**
